adafruit_requests.py

```diff
@@ -187,7 +221,9 @@ def _readto(self, first, second=b""):
 
         return b""
 
-    def _read_from_buffer(self, buf=None, nbytes=None):
+    def _read_from_buffer(
+        self, buf: Optional[bytearray] = None, nbytes: Optional[int] = None
```

Comment:
Could `buf` be something else besides a `bytearray` ?

Reply:
I think this has to be of type `bytearray` based on this line:

https://github.com/tekktrik/Adafruit_CircuitPython_Requests/blob/72a300f9e339f4c9cd2e2364517bbf9aa997474d/adafruit_requests.py#L234